code/game/objects/structures/tables_racks.dm

```diff
@@ -39,13 +43,18 @@
 			new sheet_type(src)
 	return ..()
 
-/obj/structure/table/proc/update_adjacent(location = loc)
+/obj/structure/table/proc/update_adjacent(location = loc, unflipping = FALSE)
```

Comment:
This proc doesnt make sense to me doesnt A) QUEUE_SMOOTH handle everything this proc does B) random new unused arg

Reply:
update_adjacent is still needed, tables can be flipped on their sides and when one section is destroyed it requires special handling so that the icons around do not break